shared/src/utils/stats/RecipientStatsCalculator.ts

```
@@ -1,81 +1,63 @@
+		}) as TotalRecipientsByStatus;
+
+	recipientsCountByOrganisationAndStatus = () =>
+		this.recipients
+			.groupBy('organisation')
+			.map((recipients, organisation) => ({
+				[organisation]: {
+					total: recipients.length,
+					..._(recipients)
+						.groupBy('progr_status')
+						.map((recipients, status) => ({ [status]: recipients.length }))
+						.reduce((a, b) => ({ ...a, ...b }), {}),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)